controllers/propagator/encryption.go

```diff
@@ -108,3 +111,45 @@ func (r *PolicyReconciler) getEncryptionKey(clusterName string) ([]byte, error)
 
 	return key, nil
 }
+
+// getInitializationVector retrieves the initialization vector from the annotation
+// "policy.open-cluster-management.io/encryption-iv" if the annotation exists or generates a new
+// initialization vector and adds it to the annotations object if it's missing.
+func (r *PolicyReconciler) getInitializationVector(
+	policyName string, clusterName string, annotations map[string]string,
+) ([]byte, bool, error) {
+	ivAnnotation := "policy.open-cluster-management.io/encryption-iv"
+	if initializationVector, ok := annotations[ivAnnotation]; ok {
+		log.V(2).Info("Found initialization vector annotation", "policy", policyName, "cluster", clusterName)
```

Comment:
Instead of repeating `"policy", policyName, "cluster", clusterName` in all the log messages, could you please reassign `log` at the top of the method using `log.WithValues`?